config/overlays/dev/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+# Adds namespace to all resources.
+namespace: gatekeeper-system
+
+bases:
+  - ../..
+
+patches:
```

Comment:
This is causing issues when using kustomize v3.0.3. updating to `patchesStrategicMerge` seems to fix it.